src/server/api/controllers/sensor-readings.controller.js

```diff
@@ -0,0 +1,20 @@
+const knex = require('../../config/db');
+const Error = require('../lib/utils/http-error');
+
+const getSensorReadingByMaterialId = async (materialId) => {
+  try {
+    const sensorReading = await knex('sensor_readings')
```

Comment:
```suggestion
    const sensorReading = await knex('sensors_readings')
```

Reply:
> 

I noticed the spelling mistake in table name, in today's class but you have already made a correction :) Thank you